modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,6 +16,28 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
+[[cypher-deprecations-additions-removals-5.21]]
+== Neo4j 5.21
+
+=== New features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+a|
+label:syntax[]
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+RETURN upper('abc'), lower('ABC')
+----
+| Introduction of a xref:functions/string.adoc#functions-lower[lower()] and xref:functions/string.adoc#functions-upper[upper()] function.
+These are alias' of the xref:functions/string.adoc#functions-tolower[toLower()] and xref:functions/string.adoc#functions-toupper[toUpper()] functions.
```

Comment:
I think a docs person or you (who have English as mother tongue) need to confirm if this is correct, but I would have used **aliases** instead of **alias'**